src/Types/FormTypes.ts

```diff
@@ -86,9 +86,38 @@ export type EditQuestionsAction =
   | {
       type: "EDIT_VALIDATION";
       index: number;
-      validations: any[];
+      validations: Validation;
     };
 
+export type ValidationTypeEnum =
+  | "CHECKBOX"
+  | "EMAIL"
+  | "LINK"
+  | "WORD"
+  | "CHAR";
+
+export type ValidationType = {
+  type: string;
+  default: "WORD";
+};
+
+export type ValidationMin = {
+  type: number;
+  default: 0;
+};
+
+export type ValidationMax = {
+  type: number;
+  default: 0;
+};
```

Comment:
Currently unused but I'm thinking of leaving it in since we might try and use it later? Thoughts @GodGreg

Reply:
We can leave it. The validtionTypeEnum should be used later when we expand the validation types. 

The validationMin/Max can just stay as a number.